### PR TITLE
[SM-264] Export UI changes

### PR DESCRIPTION
**Pull request by cd-bitwarden**
## Type of change

```
- [ x ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

UI did not look like it was supposed to, it ended up being that we aren't using the storybook component properly

## Code changes

Update export.component.html to use the component properly

Added a period to the messages for EN and ENG_GB

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: apps/web/src/locales/en_GB/messages.json [hunk 1]
```diff
@@ -915,7 +915,7 @@
     "message": "This file export will be password protected and require the file password to decrypt."
   },
   "exportPasswordDescription": {
-    "message": "This password will be used to export and import this file"
+    "message": "This password will be used to export and import this file."
```
Comment (djsmith85) [inline apps/web/src/locales/en_GB/messages.json:918]: Please do not edit any message.json files other than the source: en/messages.json

**Comment by Hinton** [comment]
I believe this was resolved with the #4057

**Comment by cd-bitwarden** [comment]
> I believe this was resolved with the #4057

I don't think it was fully fixed with #4057, one of the changes I made here was making sure the storybook component was correctly used 😄

**Comment by cd-bitwarden** [comment]
This has been addressed with #4057

**Pull request closed without merging**